frontend_app_html-document/test/container/HtmlDocument.spec.js

```diff
@@ -0,0 +1,220 @@
+
+  const wrapper = shallow(<HtmlDocument {...props} />)
+
+  describe('TLM handlers', () => {
+    describe('eventType content', () => {
+      describe('handleContentCreated', () => {
+        describe('create a new comment', () => {
+          it('should update the timeline if is related to the current html-document', () => {
+            const tlmData = {
+              content: {
+                ...commentTlm,
+                parent_id: contentHtmlDocument.htmlDocument.content_id,
+                content_id: 9
+              }
+            }
+            wrapper.instance().handleContentCreated(tlmData)
+            expect(wrapper.state('timeline')[wrapper.state('timeline').length - 1].content_id).to.equal(tlmData.content.content_id)
+          })
+
+          it('should not update the timeline if is not related to the current html-document', () => {
+            const tlmDataOtherContent = {
+              content: {
+                ...commentTlm,
+                parent_id: contentHtmlDocument.htmlDocument.content_id + 1,
+                content_id: 12
+              }
+            }
+            const oldTimelineLength = wrapper.state('timeline').length
+            wrapper.instance().handleContentCreated(tlmDataOtherContent)
+
+            expect(wrapper.state('timeline').length).to.equal(oldTimelineLength)
+          })
+
+          it('should sort the timeline if two TracimLiveMessages arrive in the wrong order', () => {
+            const tlmData1 = {
+              content: {
+                ...commentTlm,
+                parent_id: contentHtmlDocument.htmlDocument.content_id,
+                content_id: 10,
+                created: '2020-05-22T14:02:02Z'
+              }
+            }
+
+            const tlmData2 = {
+              content: {
+                ...commentTlm,
+                parent_id: contentHtmlDocument.htmlDocument.content_id,
+                content_id: 11,
+                created: '2020-05-22T14:02:05Z'
+              }
+            }
+
+            wrapper.instance().handleContentCreated(tlmData2)
+            wrapper.instance().handleContentCreated(tlmData1)
+            expect(wrapper.state('timeline')[wrapper.state('timeline').length - 1].content_id).to.equal(tlmData2.content.content_id)
+
+            expect(wrapper.state('timeline')[wrapper.state('timeline').length - 2].content_id).to.equal(tlmData1.content.content_id)
+
+          })
+        })
+      })
+
+      describe('handleContentModified', () => {
+        describe('modify the content name', () => {
+          const tlmData = {
+            author: contentHtmlDocument.htmlDocument.last_modifier,
+            content: {
+              ...contentHtmlDocument.htmlDocument,
+              label: 'newContentName'
+            }
+          }
+
+          before(() => {
+            wrapper.instance().handleContentModified(tlmData)
+          })
+
+          it('should be updated with the content modified', () => {
```

Comment:
what should be updated ? I don't get this test name as I read it like so:
handleContentModified
  modify the content name should be updated with the content modified

I think it should be something like `it('should change the property content of the state', () => {`

Reply:
do the same for similar tests bellow